apps/admin_panel/assets/src/omg-page-api/index.js

```diff
@@ -88,31 +96,37 @@ class ApiKeyPage extends Component {
     return (
       <KeySection>
         <h3>Admin API Key</h3>
-        {/* <p>
-          When asked, “If you could wish for one thing only, what would that wish be?” almost
-          everyone; from beauty pagent contestants, to politicians, to religious leaders, to
-          children,
-        </p> */}
+        <p>
+        The Admin API key is used to authenticate an API and allows that specific API to access various admin-related functions such as creating new minted tokens, mint more tokens, create and manage accounts, create new API keys, etc.
+        </p>
```

Comment:
What are Admin API Keys exactly? A subset of client API keys? If those are disabled, I think you should be able to create them 🤔 

As a reminder, here are the types we have:
- Client API Keys (Either to access the Admin API before logging in, or to authenticate iOS/Android non-sensitive clients)
- Access/Secret Keys (Super powerful keys used in server apps to calls the eWallet API and do pretty much anything, should never be embedded in a client!)